docs/DESIGN.md

```diff
@@ -1,67 +1,81 @@
-## Pinterest DoctorKafka Design
+# Pinterest DoctorKafka Design
 
-#### High Level Design
+## Components
```

Comment:
Can we keep the `High Level Design` section name?  When we present the design, we always go from high level design to details.